tests/test_data.py

```diff
@@ -75,17 +80,17 @@ def test_dataset(
     ds = BagDataset(
         bags=[
             [
-                make_feature_file(
+                make_old_feature_file(
                     feats=torch.rand((12, dim_feats)), coords=torch.rand(12, 2)
                 )
             ],
             [
-                make_feature_file(
+                make_old_feature_file(
                     feats=torch.rand((8, dim_feats)), coords=torch.rand(8, 2)
                 )
             ],
             [
-                make_feature_file(
+                make_old_feature_file(
                     feats=torch.rand((34, dim_feats)), coords=torch.rand(34, 2)
                 )
             ],
```

Comment:
Please turn this into a [parametrised test](https://docs.pytest.org/en/stable/how-to/parametrize.html), in order to test both the old and new feature file (i.e. pass feature_file_creator as an additional argument that is set to either `make_feature_file` or `make_old_feature_file`)